haystack/components/preprocessors/document_splitter.py

```diff
@@ -8,13 +8,12 @@
 from more_itertools import windowed
 
 from haystack import Document, component, logging
+from haystack.components.preprocessors.sentence_tokenizer import Language, SentenceSplitter, nltk_imports
 from haystack.core.serialization import default_from_dict, default_to_dict
 from haystack.utils import deserialize_callable, serialize_callable
 
 logger = logging.getLogger(__name__)
 
-# Maps the 'split_by' argument to the actual char used to split the Documents.
-# 'function' is not in the mapping cause it doesn't split on chars.
 _SPLIT_BY_MAPPING = {"page": "\f", "passage": "\n\n", "sentence": ".", "word": " ", "line": "\n"}
```

Comment:
Could we readd the note? and also maybe rename this to `_CHARACTER_SPLIT_BY_MAPPING` since we have now two options that don't appear in this mapping (nltk_sentence and function).

Reply:
sure, I just think the note is too much, and doesn't add much, I can add something shorter